packages/flutter/lib/src/widgets/navigator.dart

```diff
@@ -1852,7 +1852,7 @@ class NavigatorState extends State<Navigator> with TickerProviderStateMixin {
     assert(newRoute.overlayEntries.isEmpty);
     final Route<dynamic> oldRoute = _history.isNotEmpty ? _history.last : null;
     newRoute._navigator = this;
-    newRoute.install(_currentOverlayEntry);
+    newRoute.install(null);
```

Comment:
This doesn't look right. From the [documentation](https://master-docs.flutter.io/flutter/widgets/Route/install.html):

> The `insertionPoint` argument will be null if this is the first route inserted. Otherwise, it indicates the overlay entry to place immediately below the first overlay for this route.

This is not the first route that's inserted...

Reply:
what's the right solution to solve?